src/utils/notifications.js

```diff
@@ -67,84 +71,94 @@ const validCollectibleTransaction = (transaction: ?Object): boolean => {
-    } else if (sender === myEthAddress && status === 'confirmed') {
-      title = t('tokenValue', { value: amount, token: asset });
-      message = t('notification.transactionReceived');
-    }
-
-    result = {
-      title,
-      message,
-      asset,
-      status,
-      type: notification.type,
-    };
+    result = { type: notification.type };
   }
 
   if (notification.type === BADGE) {
-    result = {
-      title: t('notification.success'),
-      message: t('notification.newBadgeReceived'),
-      type: notification.type,
-    };
+    result = { type: notification.type };
   }
 
   if (!!notification.type && notification.type.toUpperCase() === COLLECTIBLE) {
     if (!parsedNotification || !validCollectibleTransaction(parsedNotification)) return result;
+    result = { type: COLLECTIBLE };
+  }
+
+  return result;
+};
+
+const isSameAddress = (a: ?string, b: ?string) => a && b && a.toUpperCase() === b.toUpperCase();
+
+export const getToastNotification = (data: mixed, myEthAddress: string): null | Notification => {
+  if (typeof data !== 'object') return null;
+  const { type, msg } = data ?? {};
+  const notification = typeof msg === 'string' && parseNotification(msg);
+  if (!notification) return null;
+
+  if (type === FCM_DATA_TYPE.BCX) {
+    if (!validBcxTransaction(notification)) return null;
 
-    let message = '';
-    let title = '';
     const {
-      contractName,
-    } = parsedNotification;
-    const sender = parsedNotification.fromAddress.toUpperCase();
-    const receiver = parsedNotification.toAddress.toUpperCase();
-
-    if (receiver === myEthAddress) {
-      title = contractName;
-      message = t('notification.receivedCollectible');
-    } else if (sender === myEthAddress) {
-      title = contractName;
-      message = t('notification.collectibleSentAndReceived');
+      asset,
+      status,
+      value,
+      decimals,
+      fromAddress,
+      toAddress,
+    } = notification;
+
+    const paymentInfo = t('tokenValue', {
+      value: utils.formatUnits(EthersBigNumber.from(value.toString()), decimals),
+      token: asset,
+    });
+
+    if (isSameAddress(toAddress, myEthAddress) && status === 'pending') {
+      return {
+        message: t('notification.transactionReceivedPending', { paymentInfo }),
+        emoji: 'ok_hand',
+      };
+    } else if (isSameAddress(toAddress, myEthAddress) && status === 'confirmed') {
+      return {
+        message: t('notification.transactionReceivedConfirmed', { paymentInfo }),
+        emoji: 'ok_hand',
+      };
+    } else if (isSameAddress(fromAddress, myEthAddress) && status === 'pending') {
+      return {
+        message: t('notification.transactionSentPending', { paymentInfo }),
+        emoji: 'ok_hand',
+      };
+    } else if (isSameAddress(fromAddress, myEthAddress) && status === 'confirmed') {
+      return {
+        message: t('notification.transactionSentConfirmed', { paymentInfo }),
+        emoji: 'ok_hand',
+      };
```

Comment:
Are we no longer informing about outgoing transactions?

Reply:
If you're referring to the underlying translation string -- yes, it should be "... sent." here, thanks for noticing.